### PR TITLE
[17.0][MIG] mis_builder_cash_flow

### PR DESCRIPTION
**Pull request by cormaza**
Standard Migration

### CODE REVIEW

**Review by yvaucher — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

File: mis_builder_cash_flow/views/mis_cash_flow_forecast_line_views.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        </field>
+    </record>
+    <record model="ir.ui.view" id="mis_cash_flow_forecast_line_view_tree">
+        <field name="name">mis.cash_flow.forecast_line.tree</field>
+        <field name="model">mis.cash_flow.forecast_line</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <!-- pylint:disable=duplicate-xml-fields -->
+                <field name="company_id" groups="base.group_multi_company" />
+                <field name="company_id" invisible="1" />
```
Comment (sa3m) [inline mis_builder_cash_flow/views/mis_cash_flow_forecast_line_views.xml:72]: It seems to me that invisible in tree is no longer functional. You must use column_invisible.

Comment (pedrobaeza) [reply mis_builder_cash_flow/views/mis_cash_flow_forecast_line_views.xml:72]: Yeah, that's true. cc @cormaza

**Pull request merged**